app/controllers/AsyncController.scala

```diff
@@ -20,8 +20,8 @@ object AsyncController extends Controller {
   }
 
   private def simpleZ3Check(): Future[String] = Future {
-    val config = new util.HashMap[String, String]()
-    config.put("model", "true")
+    val config = new util.HashMap[String, String]()  
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `cli`.*

The issue can be fixed by applying the following patch:

```diff
--- a/app/controllers/AsyncController.scala
+++ b/app/controllers/AsyncController.scala
@@ -20,7 +20,7 @@
   }
 
   private def simpleZ3Check(): Future[String] = Future {
-    val config = new util.HashMap[String, String]()  
+    val config = new util.HashMap[String, String]()
     config.put("model", "true")  
     config.put("unsat_core", "true")
     val context = new Context(config)
```